cmd/root.go

```diff
@@ -135,6 +133,8 @@ func init() {
 	_ = rootCmd.PersistentFlags().MarkHidden("plugin-dir")
 	_ = rootCmd.PersistentFlags().MarkHidden("policy-dir")
 
+	_ = viper.BindPFlag("verbose", rootCmd.PersistentFlags().Lookup("verbose"))
```

Comment:
Moved the binding here to have everything in the same place (we could also put each bind below each flag definition, but we should be consistent with either approach)